src/engine/Actor.ts

```diff
@@ -426,6 +426,27 @@ module ex {
        // Override me
     }
 
+    /**
+     * Gets wether the actor is Initialized 
+     */
+    public get isInitialized(): boolean {
+       return this._isInitialized;
+    }
+
+    /**
+     * Initializes this actor and all it's child actors, meant to be called by the Scene before first update.
+     */
+    public initialize(engine: Engine) {
```

Comment:
I updated our external Trello discussion, so no changes immediately needed here until we decide.